--- ../XML-LibXML-SAX-ChunkParser-0.00003/Changes	2008-10-09 17:12:40.000000000 +0900
+++ ../XML-LibXML-SAX-ChunkParser/Changes	2011-10-21 10:54:44.000000000 +0900
@@ -1,6 +1,9 @@
 Changes
 =======
 
+0.00004 - 21 Oct 2011
+  - Move to github
+
 0.00003 - 09 Sep 2008
   - Remove previous instances of this module from ParserDetail.ini
 
Only in ../XML-LibXML-SAX-ChunkParser-0.00003: diff.patch
diff -dur ../XML-LibXML-SAX-ChunkParser-0.00003/lib/XML/LibXML/SAX/ChunkParser.pm ../XML-LibXML-SAX-ChunkParser/lib/XML/LibXML/SAX/ChunkParser.pm
--- ../XML-LibXML-SAX-ChunkParser-0.00003/lib/XML/LibXML/SAX/ChunkParser.pm	2008-10-09 17:18:43.000000000 +0900
+++ ../XML-LibXML-SAX-ChunkParser/lib/XML/LibXML/SAX/ChunkParser.pm	2011-10-21 10:55:54.000000000 +0900
@@ -1,5 +1,3 @@
-# $Id: /mirror/coderepos/lang/perl/XML-LibXML-SAX-ChunkParser/trunk/lib/XML/LibXML/SAX/ChunkParser.pm 87623 2008-10-09T08:23:46.440905Z daisuke  $
-
 package XML::LibXML::SAX::ChunkParser;
 use strict;
 use base qw(XML::SAX::Base);
@@ -57,7 +55,7 @@
 =head1 SYNOPSIS
 
   local $XML::SAX::ParserPackage = 'XML::LibXML::SAX::ChunkParser';
-  my $parser = XML::SAX::ParserFactory->new(Handler => $myhandler);
+  my $parser = XML::SAX::ParserFactory->parser(Handler => $myhandler);
 
   $parser->parse_chunk($xml_chunk);
 
